frontend/src/utils/api.ts

```diff
@@ -269,7 +269,7 @@ window.api = {
           searchPath: searchParams.join('_').toLowerCase(),
           repoName: `${repo.displayName}`,
           location: `${repo.location.location}`,
-          checkedFileTypeContribution: 0,
+          checkedFileTypeContribution: -1,
```

Comment:
I am just thinking if it will be simpler to just update the type and set the initial value to undefined?